### ISSUE TITLE
decentralised introducers

### ISSUE THREAD

**Issue by kodedninja** [m-onz/dmail-paper#1]
Just a little idea, that maybe instead of 1 centralised introducer service there could be more, hosted by the community, that share data and work together. 

This way if one of them is taken down there would be no / very small negative results. I think mastodon works like this too. 

Interesting ideas, though. I hope I can contribute with some help. :v:

**Comment by m-onz** [comment]
Thank you... i briefly mention in the paper the benefits of not having a single introducer server and even having privately run services for individual groups.

Any collaboration is welcome, keep an eye on the repos I'm just about to start working on now.

Thanks for reaching out!  ✌️

**Comment by kodedninja** [comment]
I tried to bring in context more the work together part, meaning that introducer a has access to introducer b's data and all around.

Sorry if this is straight forward. :smile:

**Comment by m-onz** [comment]
Aaaaagh ok...... that is slightly different.

The introducer services could observe each other... you could have a introducer service for introducer services (fractal, turtles all the way down.. my head just exploded).

**Comment by kodedninja** [comment]
Yeah, exactly :smile: I don't know how is this done in mastodon, but it's a pretty nice structure for things like this.

**Comment by m-onz** [comment]
Thanks for the link... I need to put some more thought into it. The current version of the paper is just a draft. I'm getting bogged down with crypto stuff at the minute but I'm hoping to create an initial (probably naive and simplified) version of datmail in the next week or two.
